### PR TITLE
New test for the interop construct

### PR DESCRIPTION
**Pull request by jrreap**
Test to see if the new interop construct properly offloads and interact with a nested instruction.

### CODE REVIEW

File: tests/5.1/interop/test_interop_target.c [hunk 1]
```diff
@@ -0,0 +1,66 @@
+//===--------------------- test_interop_target.c ----------------------------===//
+//
+// OpenMP API Version 5.1 Nov 2020
+// 
+// Test for interop construct utilizing target as the interopability. Runs
+// a OpenMP target construct and treating it as an external operation to test
+// if the interop contrruct properly executes and returns the result. Test
```
Comment (spophale) [inline tests/5.1/interop/test_interop_target.c:7]: construct ?

Comment (jrreap) [reply tests/5.1/interop/test_interop_target.c:7]: It's listed as a construct in the spec, should I be using different terminology?

Comment (nolanbaker31) [reply tests/5.1/interop/test_interop_target.c]: I believe it was referring to a spelling mistake.

**Review by nolanbaker31 — APPROVED** [review]

**Review by mjcarr458 — APPROVED** [review]
Looks good

**Pull request merged**